real_estate/models/estate_property.py

```diff
@@ -0,0 +1,129 @@
+from odoo import api, exceptions, fields, models
+from dateutil.relativedelta import relativedelta
+
+
+class EstateProperty(models.Model):
+    _name = 'estate.property'
+    _description = "Property"
+
+    # Description
+    name = fields.Char(
+        string="Property Name",
+        required=True)
```

Comment:
```suggestion
    name = fields.Char(string="Property Name", required=True)
```
It can be in one consist only 62 char. Please make standard if the line crosses 79/99 char in single then only make it into multi-line (usually 99 Char is fine)